include/smb2/smb2.h

```diff
@@ -19,7 +19,11 @@
 #ifndef _SMB2_H_
 #define _SMB2_H_
 
+#ifdef __APPLE__
+#include <smb2-errors.h>
+#else
 #include <smb2/smb2-errors.h>
+#endif
```

Comment:
This seems like something wrong with the setup for the apple compiler. It should use both -I include and -I include/smb2
so that smb2-errors.h should always be found regardless of if <smb2-errors.h> or <smb2/smb2-errors.h> are used.

Reply:
Apple xcode normally would have some stuff like including in one dir.